datacube_apps/ncml.py

```diff
@@ -0,0 +1,190 @@
+"""
+Create statistical summaries command
```

Comment:
Don't hold off on merging, but it would great to have a much longer description about what this tool is and what it is used for.